docs/zkapps/tutorials/12-cross-contract-calls.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: 'Tutorial 12: Cross Contract Calls'
+hide_title: true
+sidebar_label: 'Tutorial 12: Cross Contract Calls'
+description: Guided steps to learn about the ways to interact with a smart contract from another smart contract.
+keywords:
+  - smart contracts
+  - zkapps
+  - zero knowledge proof programming
+  - zk proof
+  - zk
+  - cross contract call
+  - mina
+---
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+
+# Tutorial 12: Cross Contract Calls
+
+In this tutorial, you learn how smart contracts on a blockchain can interact by calling functions in each other's code, enabling building modular and complex decentralized applications.
+
+Cross contract calls allow smart contracts on a blockchain to interact with each other. This enables building complex decentralized applications (dApps) from multiple modular components. In a cross contract call, a function in one smart contract can call a function in another smart contract to leverage existing code and functionality.
```

Comment:
```suggestion
Cross contract calls allow smart contracts on a blockchain to interact with each other. This enables the building of complex decentralized applications (Dapps) from multiple modular components. In a cross-contract call, a function in one smart contract can call a function in another smart contract to leverage existing code and functionality.
```

Reply:
use Dapp for the capitalization (yes, it's different than others) https://github.com/o1-labs/docs2/wiki/Word-list#dapp